### PR TITLE
features: Add link to translation docs

### PR DESCRIPTION
**Pull request by prtksxna**

### CODE REVIEW

File: content/pages/features.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ sortorder: 20
   <div class="col-lg-6">
     <h2>Available in your language</h2>
     wallabag is translated into more than ten languages.<br />
-    If yours is not yet available, please help us to do it. 
+    If yours is not yet available, please help <a href="http://doc.wallabag.org/en/master/developer/translate.html">translate</a>.
```
Comment (nicosomb) [inline content/pages/features.md]: `please help` or `please help us` ?

Comment (prtksxna) [inline content/pages/features.md]: Both are correct. Could add back the  `us`, if you'd prefer.

Comment (nicosomb) [inline content/pages/features.md]: I'm not so fluent in english, I prefer ask ;-)

**Comment by nicosomb** [comment]
Thank you!

File: content/pages/features.md (hunk 1 above)
Comment (prtksxna) [inline content/pages/features.md]: Good thinking! I'll remember to mention alternatives next time I submit a PR. :)

**Pull request merged**